programmers/난이도별/level01.제일_작은_수_제거하기/SSSOy.java

```diff
@@ -0,0 +1,26 @@
+import java.util.*;
+
+class Solution {
+    public int[] solution(int[] arr) {
+        
+        int min = arr[0];
```

Comment:
```suggestion
        int min = Arrays.stream(arr).reduce(Integer.MAX_VALUE, Math::min); 
```
이런식으로 최솟값을 바로 구할 수 있습니다!